ui/library/src/context/ScrollContext.ts

```diff
@@ -66,6 +67,12 @@ const DEFAULT_CONTEXT: IScrollContext = {
   scrollToPage: opts => {
     logProviderWarning(`scrollToPage(${JSON.stringify(opts)})`, 'ScrollContext');
   },
+  recalibrateScrollPosition: zoomMultiplier => {
```

Comment:
@ericmarsh995 not sure if i think recalibrate is a right name but maybe updateScrollPosition 🤔 since technically we update when our zoom scale change but thats just my opinion